### DISCUSSION TITLE
Topicref with mailto: and scope="peer"

### DISCUSSION THREAD

**Discussion by jlacour31** [dita-ot/dita-ot#3944] [Q&A] [answered]
We have a test in Oxygen asserting that the following structure:
`<topicref href="mailto:support@oxygenxml.com" scope="peer" format="html">`
Do not generates an extra target='_blank' attribute in HTML.

Unfortunately after updating the DITA-OT version to 3.7.1, this test fails with:
> debug-filter:
>    [filter] Using Xerces grammar pool for DTD and schema caching.
>    [filter] Processing file:/.../test.ditamap
>    [filter] java.lang.RuntimeException: Expected scheme-specific part at index 7: mailto:
>    [filter] 	at org.dita.dost.util.URLUtils.setQuery(URLUtils.java:577)
>    [filter] 	at org.dita.dost.writer.ValidationFilter.validateReference(ValidationFilter.java:202)
>    [filter] 	at org.dita.dost.writer.ValidationFilter.startElement(ValidationFilter.java:97)

Did this behavior changes again in 3.7.2 or do we need to udpate the original topicref in the ditamap?

I've already checked that if the scope is changed to external, the error disappear but the target attribute appears.

Also, even if the @target attribute is present, a click on the link doesn't open a new tab in browser but the mail client directly (or at least ask where to open the link).

**Comment by chrispy-snps** [comment, marked as answer]
@jlacour31 - in 3.7.1, I get the same Java failure:

```
$ ./dita-ot-3.7.1/bin/dita -i map.ditamap -f html5
Error: java.lang.RuntimeException: Expected scheme-specific part at index 7: mailto:
```

In 3.7.3, I get DITA-OT errors but no Java failure:

```
$ ./dita-ot-3.7.3/bin/dita -i map.ditamap -f html5
[move-meta] file:/mnt/c/nobackup/mailto/map.ditamap:6:77: [DOTX020E][ERROR]: Missing navtitle attribute or element for peer topic "mailto:support@oxygenxml.com". References must provide a local navigation title when the target is not a local DITA resource.
[move-meta] file:/mnt/c/nobackup/mailto/map.ditamap:6:77: [DOTX024E][ERROR]: Missing linktext and navtitle for peer topic "mailto:support@oxygenxml.com". References must provide a local navigation title when the target is not a local DITA resource.
```

so maybe your testcase behaves better in 3.7.3 too?

**Reply by jlacour31** [reply]
@chrispy-snps I updated to DITA-OT 3.7.3 and the problem is not present anymore. I will consider this problem as resolved.
